e2e/app.po.ts

```diff
@@ -195,6 +197,16 @@ export class AppPage {
     }).then(jsonString => JSON.parse(<string> jsonString));
   }
 
+  waitForElementToBePresent(element) {
+    browser.wait(function () {
+        return element.isPresent();
+    }, 60000);
+
+    browser.wait(function () {
+        return element.isDisplayed();
+    }, 60000);
+  }
+
```

Comment:
Sorry, I didn't notice we have functionality for that. 
What about that https://github.com/lvydra/syndesis-e2e-tests/commit/594ea73f0f897cb5dd0b56bf262e7990b3b8a9d1